eng/restore/harvestPackages/harvestPackages.props

```diff
@@ -131,7 +131,7 @@
   <Version>4.5.2</Version>
 </PackageReference>
 <PackageReference Include="System.Runtime.WindowsRuntime">
-  <Version>4.3.0</Version>
+  <Version>4.6.0-preview8.19405.3</Version>
```

Comment:
cc @ericstj I believe we won't want to change this

Reply:
Actually its not being used anywhere. i will rrevert it.